src/python/pants/rules/core/test.py

```diff
@@ -50,9 +58,22 @@ def is_testable(
     return is_valid_target_type and has_sources
 
 
+@dataclass(frozen=True)
+class AddressAndDebugResult:
+  address: BuildFileAddress
+  test_result: TestDebugResult
+
+
 @console_rule
-async def fast_test(console: Console, addresses: BuildFileAddresses) -> Test:
-  results = await MultiGet(Get(AddressAndTestResult, Address, addr.to_address()) for addr in addresses)
+async def fast_test(console: Console, options: TestOptions, addresses: BuildFileAddresses) -> Test:
```

Comment:
Adding `TestOptions` here fails some existing tests using the `run_rule` test helper, and I'm not familiar with how to best mock out these options in the current test infra - [I reached out on Slack](https://pantsbuild.slack.com/archives/C0D7TNJHL/p1576525353020000) and could use a hand in putting together the appropriate tests for this change.

Reply:
`s/fast_test/test`.

If testing is still an issue, I can pair with you this week or next :)

Reply:
Bump on renaming `fast_test` to `test`.

Reply:
I ran into issues with pytest when changing this to `test` - as discussed on Slack, I've updated it to `run_tests` instead as a better fit than `fast_test`.